pkg/features/kube_features.go

```diff
@@ -172,6 +172,11 @@ const (
 	// Disable in-tree functionality in kubelet to authenticate to cloud provider container registries for image pull credentials.
 	DisableKubeletCloudCredentialProviders featuregate.Feature = "DisableKubeletCloudCredentialProviders"
 
+	// owner: @AxeZhan
```

Comment:
we use to tie feature gates that go through all the stability stages to KEPs so we can have a way to track them

Reply:
We haven't decide if we are going to use the feature gate yet.
But is this a must-to-do?  I don't think this change worth a KEP.